File: ejercicio1.py
#se ide numero de casos
n=int(input("Casos de prueba: "))
i=0     #aterrizaje de contador
i2=0    #aterrizaje de contador secundario
tot=0   #aterrizaje de variable clave
while (i<n):    #ciclo while primario, se repite n veces
    i=i+1   #contador
    
    #se le pide al usuario el rango en el caso i
    a=int(input("inicio de rango "+str(i)+": "))
    b=int(input("fin de rango "+str(i)+": "))
    
    #linea que se ejecuta si el numero de inicio de rango es par
    if (a%2==0):
        a=a+1
        
    #se le asigna el valor de a al contador secundario    
    i2=a
    while (i2<b):   #ciclo que se repite segun la cantidad de numeros impares dentro de un rango i
        i2=i2+2     #contador secundario
        tot=tot+a   #acumulador
        a=a+2       
    i2=0
    print("la suma de los numeros impares en el rango ",i,"es: ",tot)
    tot=0
        
        

File: ejercicio1.cpp
#include <iostream>
using namespace std;

int main(){
    int tot=0;
    int n=0;
    cout << "Numero de casos de prueba: " << endl;
    cin >> n;
   //procedimiento que se repita segun el numero de casos que se especifique
    for(int i=1;i<=n;i++){
        //reinicializacion de valores 
        int a=0;
        int b=0;
        //solicitud de valores
        cout << "Caso "<<i<<" :"<<endl;
        cout << "Inicio de rango "<<i<<" :"; cin >> a;
        cout << "" << endl;
        cout << "Final de rango "<<i<<" :"; cin >> b;
        cout << "" << endl;
        //parametro que convierte el numero a impar en el caso de que el primer numero sea par
        if (a%2==0){
            a=a+1;
        }
        //ciclo de acumuladores 
        for (int j = a; j<b; j=j+2){
            tot=tot+a;
            a=a+2;
        }

        cout<<tot<<endl;
        
        tot=0;
    }
return 0;
}